bin/build-image.sh

```diff
@@ -32,53 +29,42 @@ GENERAL_FLAGS=(
     --name zwift
     --security-opt label=type:container_runtime_t
     --device /dev/dri
-    --hostname $HOSTNAME
+    --hostname "$HOSTNAME"
 
-    -e DISPLAY=$DISPLAY
-    -e XAUTHORITY=$XAUTHORITY
-    -e CONTAINER=$CONTAINER_TOOL
+    -e DISPLAY="$DISPLAY"
+    -e XAUTHORITY="$XAUTHORITY"
+    -e CONTAINER="$CONTAINER_TOOL"
 
     -v /tmp/.X11-unix:/tmp/.X11-unix
-    -v /run/user/$UID:/run/user/$ZWIFT_UID
+    -v /run/user/$UID:"/run/user/$ZWIFT_UID"
 )
 
-
 # Check for proprietary nvidia driver and set correct device to use
-if [[ -f "/proc/driver/nvidia/version" ]]
-then
-    VGA_DEVICE_FLAG="--gpus all"
+if [[ -f "/proc/driver/nvidia/version" ]]; then
+    VGA_DEVICE_FLAG=(--gpus all)
 else
-    VGA_DEVICE_FLAG="--device /dev/dri:/dev/dri"
+    VGA_DEVICE_FLAG=(--device /dev/dri:/dev/dri)
 fi
 
 # Initiate podman Volume with correct permissions
-if [[ "$CONTAINER_TOOL" == "podman" ]]; then 
+if [[ "$CONTAINER_TOOL" == "podman" ]]; then
     # Add ipc host to deal with an SHM issue on some machines.
-    PODMAN_FLAGS=(
-        --userns keep-id:uid=$ZWIFT_UID,gid=$ZWIFT_GID
-    )
+    PODMAN_FLAGS=(--userns "keep-id:uid=$ZWIFT_UID,gid=$ZWIFT_GID")
 fi
 
 # Cleanup on error
 trap cleanup ERR
 cleanup()
 {
-  $CONTAINER_TOOL container rm zwift
-  exit
+    $CONTAINER_TOOL container rm zwift
+    exit
 }
 
-
-$CONTAINER_TOOL build --force-rm -t $BUILD_NAME $SCRIPT_DIR/../.
-$CONTAINER_TOOL run ${GENERAL_FLAGS[@]} \
-    $VGA_DEVICE_FLAG \
-    ${PODMAN_FLAGS[@]} \
-    $IMAGE:latest \
-    $@
-
-
-$CONTAINER_TOOL commit zwift $BUILD_NAME:latest
+$CONTAINER_TOOL build --force-rm -t "$BUILD_NAME" "$SCRIPT_DIR/../."
+$CONTAINER_TOOL run "${GENERAL_FLAGS[@]}" "${VGA_DEVICE_FLAG[@]}" "${PODMAN_FLAGS[@]}" "$IMAGE:latest" "$@"
```

Comment:
I feel that this is less reabable now
```suggestion
$CONTAINER_TOOL run "${GENERAL_FLAGS[@]}" \
    "${VGA_DEVICE_FLAG[@]}" \
    "${PODMAN_FLAGS[@]}" \
    "$IMAGE:latest" \
    "$@"
```

maybe we can tweak or disable some rules so that it doesn't warn about multiple lines like this?

Reply:
That was actually me who put that on a single line since I had to modify that line to add quotes and figured it was a short line anyway. The linter does not complain about line breaks. But since you prefer to see that command in multiple lines, I split it again in 6bd0c2e.